custom/icds_reports/reports/issnip_monthly_register.py

```diff
@@ -25,6 +28,7 @@ def filter_by(self, config, column, default=0):
         )
 
     @property
+    @memoized
```

Comment:
Can you change this to use `cached_property` https://docs.djangoproject.com/en/1.11/ref/utils/#django.utils.functional.cached_property

Reply:
@emord updated